www/md_base/src/styles/styles.less

```diff
@@ -0,0 +1,11 @@
+// Import bootstrap and font awesome stylesheets
+/* base css for angular, dont display anything until angular is running */
+
+@import (inline) "../../libs/angular-material/angular-material.css";
+
+@import "./sidenav.less";
+@import "./topbar.less";
+
+[ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
```

Comment:
hmm... this line is generated by guanlecoja. I think only `ng-cloak` is using. Change it soon.